### ISSUE TITLE
how to get the selected item?

### ISSUE THREAD

**Issue by slavugan** [LeaVerou/awesomplete#16438]
how can I get the selected item in awesomplete-select or awesomplete-selectcomplete handler? I have complex item like 
<code>
<li>
 <span class='name'>blabla</span>
 <span class='email'>bla@bla</span>
</li>
</code>

and I need to get the selected item from list to create custom datapasting

**Comment by LeaVerou** [comment]
I think you need to take a look at `replace` here: http://leaverou.github.io/awesomplete/#extensibility

(but excellent point re: the event. We need to include this as an event property)

**Comment by matanox** [comment]
I think checking the underlying html element's value would be at par with using `replace`, but ultimately including the selection in the events may really seem to make sense...

**Comment by vlazar** [comment]
@slavugan Do you want to get selected DOM element `<li>` or selected item data like `{ name: "blabla" , email: "bla@bla" }`?

**Comment by vlazar** [comment]
Separate `label/value` is landed #16866
It is also possible to initialize list with separate `label/value` via `<datalist>` or `<ul>`.

Read about different `label` and `value` at the end of [Basic usage](http://leaverou.github.io/awesomplete/#basic-usage) section and about new `data` method at the end of [Extend](http://leaverou.github.io/awesomplete/#extensibility) section.

**Comment by vlazar** [comment]
Selected suggestion item is available as `text` event property for `awesomplete-highlight`, `awesomplete-select` and `awesomplete-selectcomplete` events.

``` javascript
Awesomplete.$.bind(input, {
  "awesomplete-selectcomplete": function(evt) {
    console.log(evt.text); // String {label: "Java", value: "Java", valueOf: function, toString: function}
  }
});
```
